src/components/Tab/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './Tabs.scss';
+import classNames from 'classnames';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: Tab;
+  onTableSelected: (tabId: string) => void;
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const { tabs, onTableSelected, selectedTabId } = props;
+
+  return (
+    <>
+      <div className="card-link">
+
+        {tabs.map((tab) => {
+          return (
+            <div key={tab.id}>
+              <button
+                type="submit"
+                className={classNames(
+                  'nav-link',
+                  'btn',
+                  {
+                    active: tab.id === selectedTabId.id,
+                  },
+                )}
+                onClick={() => {
+                  if (tab.id) {
```

Comment:
here you just check if tab.id in a logical sence - true. And if it will be the same id the callback will be still called. You need to check if id of current tab !== id of pressed tab